ui/pages/index.html

```diff
@@ -14,9 +14,7 @@
 </head>
 
 <body>
-    <div id="loader"></div>
```

Comment:
why?

Reply:
Since I ported cView and it has its own loading logic, I thought it is better to remove the loader from the html and just control it on our views